xml/System.Numerics/BigInteger.xml

```diff
@@ -2861,10 +2942,31 @@ value Mod 2 = 0
         <Parameter Name="value" Type="System.Numerics.BigInteger" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to convert to a <see cref="T:System.Double" />.</param>
+        <summary>Defines an explicit conversion of a <see cref="T:System.Numerics.BigInteger" /> object to a <see cref="T:System.Double" /> value.</summary>
+        <returns>An object that contains the value of the <paramref name="value" /> parameter.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The overloads of the <xref:System.Numerics.BigInteger.op_Explicit(System.Decimal)~System.Numerics.BigInteger> method define the types to which or from which a <xref:System.Numerics.BigInteger> object can be converted. Language compilers do not perform this conversion automatically because it can involve data loss. Instead, they perform the conversion only if a casting operator (in C#) or a conversion function (such as `CType` or `CDbl` in Visual Basic) is used.   
+
+ Because the <xref:System.Numerics.BigInteger> value can be outside the range of the <xref:System.Double> data type, this operation is a narrowing conversion. If the conversion is unsuccessful, it does not throw an <xref:System.OverflowException>. Instead, if the <xref:System.Numerics.BigInteger> value is less than <xref:System.Double.MinValue>, the resulting <xref:System.Double> value is <xref:System.Double.NegativeInfinity>. If the <xref:System.Numerics.BigInteger> value is greater than <xref:System.Numerics.BigInteger>, the resulting <xref:System.Double> value is <xref:System.Double.PositiveInfinity>.
```

Comment:
Two issues here:
- The xrefs to Double.Minvalue, Double.MaxValue (which isn't included yet -- see the next point), Double.PositiveInfinity, and Double.NegativeInfinity should all be `?displayProperty=fullName`.
This should be "greater than `<xref:System.Double.MaxValue?displayProperty=fullName>` instead of `<xref:System.Numerics.BigInteger>`.